solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -1,3 +1,30 @@
+.liquidity-banner {
+  margin: 0.8rem 3.2rem;
+  padding: 0.8rem 0;
```

Comment:
I think we can remove `padding`. The `Banner` component provides the right padding. According to the Figma views, I believe custom padding is no needed here.